components/03-modules/dashboard-form/dashboard-form.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="dashboard-form {{ class }}">
+    <h5 class="heading heading--fifth-level dashboard-form__title">
+        {{ title }}
+    </h5>
+
+    <div class="dashboard-form__content {{ contentClass }}">
+        {{#each fields }}
+            {{> '@input' input }}
+        {{/each}}
+
+        {{#each textareas }}
+            {{> '@input--textarea' textarea }}
+        {{/each}}
+
+        {{#each selects }}
+            {{> '@select' select }}
+        {{/each}}
+
+        {{#each checkboxes }}
+            {{> '@checkbox' checkbox}}
+        {{/each}}
+
+        <div class="dashboard-form__fields dashboard-form--hidden">
+            <h5 class="heading heading--sixth-level dashboard-form__title-form">
+                {{ formTitle }}
+            </h5>
```

Comment:
This element `h5` inside hidden element is for what reasons?

Reply:
It's just title and functionality like in standard dashboard in `account info` tab. You can see a thing like this here.

Reply:
ok, add script for hidden fields